packages/nova-routing/lib/server/routing.jsx

```diff
@@ -40,6 +40,7 @@ Meteor.startup(() => {
     },
     wrapperHook(req, res, appGenerator) {
       const { apolloClient, store } = getRenderContext();
+      store.reload({ message: 'reload store before render' });
```

Comment:
because i am afraid that in the future some other-people-custom-package may want hack/add other reducers to store before render it. yes currently SSR only use apolloClient.middleware. this code just i use to ensure reload store before render it. because sometimes i may forgot reload store after added a reducer. so, yes it can be delete